packages/vite/src/node/index.ts

```diff
@@ -39,7 +39,8 @@ export type {
   DepOptimizationOptions,
   DepOptimizationResult,
   DepOptimizationProcessing,
-  OptimizedDepInfo
+  OptimizedDepInfo,
+  OptimizedDeps
```

Comment:
The compiler complains when I don't export it.

Reply:
Ah well I can reproduce that too 🥲 Looks like an issue in api-extractor still needing the types exported even after we marked `server._optimizedDeps` as `@internal`. I'm not sure what the solution is yet, but it would be great not to export the type for internal stuff.